doc/nrf/templates/library_template_README.rst

```diff
@@ -0,0 +1,143 @@
+.. _library_template:
+
+Template: Library
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. note::
+   Use this template to create pages that describe the libraries included in |NCS|, following these instructions:
+
+   * Use this first section to give a short description (1 or 2 sentences) of the library, indicating what it does and why it should be used.
+
+   * Give the library a concise name that also corresponds to the folder name.
+
+     If the library targets a specific protocol or device, add it in the title before the library name (for example, "NFC:" or "nRF9160:").
+     Do not include the word "library" in the title.
+     Use the provided stock phrases and includes when possible.
+
+   * Sections with ``*`` are optional and can be left out.
+
+     All the other sections are required for all libraries.
```

Comment:
I would move this one line next to the line 21.

Reply:
fixed